app/src/main/java/org/inspire/breath/activities/PneumoniaSeveritySigns.java

```diff
@@ -0,0 +1,30 @@
+package org.inspire.breath.activities;
+
+import android.content.Intent;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.View;
+
+import org.inspire.breath.R;
+
+public class PneumoniaSeveritySigns extends AppCompatActivity {
```

Comment:
The results of these questions need to be stored in a BreathTestResult instance and written back to the DB, given this happens last in the flow